src/components/App/Dashboard/Balances/BalanceItem/BalanceItem.js

```diff
@@ -17,31 +18,56 @@ class BalanceItem extends Component {
   static propTypes = {
     amount: PropTypes.number,
     asset: PropTypes.string,
+    history: PropTypes.arrayOf(
+      PropTypes.shape({
+        close: PropTypes.number.isRequired,
+        high: PropTypes.number.isRequired,
+        low: PropTypes.number.isRequired,
+        open: PropTypes.number.isRequired,
+        time: PropTypes.number.isRequired,
+        volumefrom: PropTypes.number.isRequired,
+        volumeto: PropTypes.number.isRequired,
+      })
+    ).isRequired,
```

Comment:
I saw
<img width="781" alt="screen shot 2018-10-12 at 01 17 20" src="https://user-images.githubusercontent.com/59687/46837057-9a3c7800-cdbc-11e8-9952-63b365964b17.png"> that exception on my console.